js/data.js

```diff
@@ -936,6 +1112,17 @@ var data = {
         dozent: ["Dr. rer. nat. Harald Sack", "Jörg Waitelonis", "Nadine Steinmetz", "Magnus Knuth"],
         nameLV: "Semantic Multimedia"
     },
+    semanticMultimedia2: {
+        kurz: "Semantic Multimedia<br />(BPET)",
```

Comment:
I wonder whether this actually is an error, because this seminar is much more HCT than BPET, but anyways. The document says it so ...